ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -199,6 +199,362 @@ def _grpc_channel_pool(self) -> GrpcChannelPool:
         return self._session_management_client._grpc_channel_pool
 
 
+class MultiplexerSessionHandler(_BaseSessionContainer):
+    """Manages multiplexer session information."""
+
+    def __init__(
```

Comment:
Are you planning to make the `MultiplexerSessionHandler` class public? If so, please make its constructor enforce the feature toggle as well. If the feature toggle decorator doesn't work on `__init__` methods, then I guess you can call `THE_FEATURE_TOGGLE._raise_if_disabled()`. (Consider making this method public from the feature toggle class if you do.)

BTW, even if users can't construct instances of the class directly, if you return it to the user then you should make it or a corresponding ABC or Protocol public so that users can write type hints for their code.

Reply:
The last part is addressed in the next PR

Reply:
Sure. I've made the `MultiplexerSessionContainer` (previously `MultiplexerSessionHandler`) class public.